### DISCUSSION TITLE
Advice to make good LoRA model or SDXL model

### DISCUSSION THREAD

**Discussion by kukovinetsyevhenii** [bmaltais/kohya_ss#2726] [Q&A]
I have 50+ design patterns of classroom
I am trying to generate 'after' images based on 'before' images and text prompt and design pattern.
To generate good quality images, my gut tells I need to use LoRA fine tunning.
For ex: )
design pattern 1: 1 Personal Space
And the design pattern images: 
![deimages_98159_2_diverse_middle_school_student_are_sitting_on_7451dd5f-71cc-4990-9672-a0f060277653_0](https://github.com/user-attachments/assets/6cfde115-b747-4966-b237-b1b1d4afa018)
![deimages_98159_A_contemporary_middle_school_classroom_with_th_25122f99-a8df-42e6-8369-2aa2643b7296_1](https://github.com/user-attachments/assets/929ea767-c10a-444b-b98b-9b2aaf7ed289)
![deimages_98159_A_diverse_group_of_high_school_students_in_a_s_4b731258-48fa-45f7-a6af-81ad8ca3db58_1](https://github.com/user-attachments/assets/723bf2ea-0597-41e8-b473-387a6b254597)
![deimages_98159_a_white_male_and_a_hispanic_female_middle_scho_7a3d3ed7-b028-47c3-ace4-70a85b661a66_0](https://github.com/user-attachments/assets/2d3628b2-790a-4dac-8060-c8ffcdb023fc)
![deimages_98159_httpss mj runCOG-OOURc7M_A_single-line_sketch__143ec8de-f45b-45e9-84d6-8a8fc5a9775d_1](https://github.com/user-attachments/assets/2cd81aec-7bf1-49cc-b9fb-5fe52c2a050c)
![deimages_98159_lifelike_facial_features _One_quadrant_of_a_co_cf740cf7-194e-4d5a-b9b7-c03c19570d09_0](https://github.com/user-attachments/assets/db199e0b-8ff5-4c8a-b667-e018780b6975)

2:Collaborative Space
![deimages_98159_4_diverse_middle_school_students_sitting_at_a__da219395-50c2-464e-a6eb-8c9a4b058295_1](https://github.com/user-attachments/assets/e5fb680c-34f6-46a2-b422-e271b3de8737)
![deimages_98159_A_single_line_black_and_white_line_drawing_wit_7eafd042-24fd-42d6-aee4-f2a141e2fa37_2](https://github.com/user-attachments/assets/25a169d5-9fd1-4424-8364-bba58f6c1ae2)
![deimages_98159_A_single-line_black_sketch_with_a_light_waterc_87058186-5f3b-4d2f-b5f3-0bf4586533d6_0](https://github.com/user-attachments/assets/9f57f982-7252-4283-90c2-3d1165303f8f)
![deimages_98159_httpss mj runXPjcPych4YA_httpss mj runaYts49xR_d25ceaf4-d187-4b09-9cde-5bb8cc94e4de_0](https://github.com/user-attachments/assets/f15940e0-bc57-4b68-b0d9-31e694ac8133)

3 Active Space
![deimages_98159_1_diverse_middle_school_student_doing_a_handst_1a578b08-cd1d-4d06-8969-1aa89a41629b_1](https://github.com/user-attachments/assets/30b089da-f199-4094-8be9-2c97410ab1d9)
![deimages_98159_a_naturally_lighted_elementary_school_classroo_62c9b8d7-6d94-4ed7-b9d6-ce9386c222bc_0](https://github.com/user-attachments/assets/52d5fd5d-9a95-4524-b058-d0321193fd56)
![deimages_98159_A_single_line_black_and_white_line_drawing_wit_962d9710-9156-43db-9f18-7b4b729b5b17_1](https://github.com/user-attachments/assets/26c71ed0-5842-4a1a-a2bc-78246288a239)

That is for generating images of environment so we need to take care of humans as well as other objects like desk, table, walls and so on.

Here I also have good quality furniture images I think it can help us to generate better output.
![Gressco HABA Pro Breeze upp Folding Storage Trolley _ 011 Student Storage](https://github.com/user-attachments/assets/19527319-7432-4fb0-af06-60624017d4cd)
![KI Strive Nesting Chair _ 01 Personal _ 02 Collaborative](https://github.com/user-attachments/assets/eda35544-8abb-4dc4-9b1b-61363842004a)
![MiEN SBM Pneumatic Side Table _ 03 Active Space](https://github.com/user-attachments/assets/2aa45b05-e943-4611-9ae8-c41505bb9fb8)
![Muzo Nook Lounge Chairs Gray _ 01 Personal Space _ 02 Collaborative Space](https://github.com/user-attachments/assets/813f8f5b-33cf-4a38-affb-f0078df99565)

Here I want to know how to use those dataset to train LoRA or what ? 
I also want to know how to generate good quality images 

Thanks

**Comment by J-Cott** [comment]
https://www.youtube.com/results?search_query=how+to+train+an+SDXL+lora
